src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+
+const todoList = document.querySelector('.items');
+const todos = [
+  {
+    description: 'Take a walk ',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Read a book',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Wash the car',
+    completed: false,
+    index: 2,
+  },
+];
```

Comment:
- [ ] **Required:** As earlier pointed out by the previous reviewer, your code needs to display in the order of the index.

![image](https://user-images.githubusercontent.com/27709832/155612525-de8769e3-567d-478d-9e07-e7de00b183d3.png)

_**Hint:**_ You can sort the array of object to achieve this. For example `Array.sort((a, b) => a.index -b.index);`, then display them.

_If you have issues, chat me up on slack for some help._